pkg/hub/controllers/certrotation_controller.go

```diff
@@ -39,10 +38,10 @@ type certRotationController struct {
 func NewCertRotationController(
 	namespace string,
 	kubeClient kubernetes.Interface,
+	serviceIP string,
```

Comment:
could we use service domain name to replace IP?

Reply:
Yes